outbound/hmail.js

```diff
@@ -300,7 +302,7 @@ class HMailItem extends events.EventEmitter {
         const mx = this.mxlist.shift();
         const host = mx.exchange;
 
-        if (!obc.cfg.local_mx_ok) {
+        if (mx.default && !obc.cfg.local_mx_ok) {
```

Comment:
I'm trying hard to like the name `default` here. It's not very self documenting and requires one to go figure out how `default` got set. In this case, default means "looked up via DNS". But that's a bit long for a name. Perhaps something like `from_dns` or `via_dns`, so the `if` reads something like "if the mx was looked up in DNS and ! local_mx_ok)". That'll also avoid namespace collisions when someone goes searching to find how that property got set.

Reply:
I don't like default either, naming is hard...